.github/workflows/contrib-tests.yml

```diff
@@ -44,11 +44,15 @@ jobs:
         run: |
           pip install qdrant_client[fastembed]
       - name: Install unstructured when python-version is 3.9 and on linux
-        if: matrix.python-version == '3.9' && matrix.os == 'ubuntu-latest'
         run: |
           sudo apt-get update
           sudo apt-get install -y tesseract-ocr poppler-utils
           pip install unstructured[all-docs]==0.13.0
+      - name: Install and Start PostgreSQL
+        runs-on: ubuntu-latest
+        run: |
+          sudo apt install postgresql -y
+          sudo service postgresql start
```

Comment:
How about adding a python version condition like 3.11?
Also install the retrievechat-pgvector option in this section?
And update the installation section for qdrant?

Reply:
I didn't add python condition because pgvector should support Python 3.9-3.11. Let me know if you have a suggestion.

I'm not quite sure what you mean on the second question? Would you mind making a suggestion please?

I updated the installation guide for qdrant to include the proposed changes in setup.py.

Reply:
Similar to the other question, there is a section that installs qdrant dependency. With the updated setup.py, shall we use the newly added retrievechat-qdrant option?

Reply:
Yes, I have updated the notebook for agentchat_qdrant_RetrieveChat.ipynb which includes the installation instructions for qdrant. 

The agentchat_pgvector_RetrieveChat.ipynb shows how to install for pgvector. 

The Optional-Dependencies.md shows how to install all three versions. Please let me know if there are other installation instructions that need to be updated.

Reply:
Line 45 :)

Reply:
Thank you, I have made the adjustment!